packages/sdk-redux/src/reduxSlices/rtkQuery/rpcApiSlice/endpoints/flowEndpoints.ts

```diff
@@ -36,19 +36,19 @@ export const createFlowEndpoints = (builder: RpcEndpointBuilder) => ({
     }),
     flowUpdate: builder.mutation<TransactionInfo, FlowUpdateMutation>({
         queryFn: async (arg, queryApi) => {
-            const signer = await getSigner(arg.chainId);
             const framework = await getFramework(arg.chainId);
             const superToken = await framework.loadSuperToken(arg.superTokenAddress);
-            const senderAddress = arg.senderAddress ? arg.senderAddress : await signer.getAddress();
+            const senderAddress = arg.senderAddress ? arg.senderAddress : await arg.signer.getAddress();
```

Comment:
Again minor thing but imo easier to read:
const senderAddress = arg.senderAddress || await arg.signer.getAddress();